index.js

```diff
@@ -128,7 +128,8 @@ module.exports = {
   lintTree: function(type, tree) {
     // Skip if useLintTree === false.
     if (this.options['ember-cli-qunit'] && this.options['ember-cli-qunit'].useLintTree === false) {
-      return tree;
+      // Fakes an empty broccoli tree
+      return { inputTree: tree, rebuild: function() { return []; } };
```

Comment:
This is compat with ember-cli 0.2.2+, but will not work for Broccoli 1.0.0 (where the `.rebuild` API is deprecated).  

We will need to bump minors and just `return false;` at some point to avoid deprecations there (definitely fine to do at a later date when we use Broccoli 1.0.0 in ember-cli).